gradle/publish.gradle

```diff
@@ -47,19 +48,19 @@ tasks.withType(AbstractPublishToMaven).all {
     onlyIf { isAvailableForPublication(publication) }
 }
 
+def sonatypeUser = project.hasProperty('sonatypeUser') ? project.property('sonatypeUser') : System.getenv('SONATYPE_USER')
+def sonatypePassword = project.hasProperty('sonatypePassword') ? project.property('sonatypePassword') : System.getenv('sonatypePassword')
```

Comment:
Env name for password inconsistent with username

Reply:
Thanks, got it! Will switch to env variables with consistent naming